.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: "Run tests"
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:11-alpine
+        ports:
+          - "5432:5432"
+        env:
+          POSTGRES_DB: rails_test
+          POSTGRES_USER: rails
+          POSTGRES_PASSWORD: password
+
+    env:
+      RAILS_ENV: test
+      DATABASE_URL: "postgres://rails:password@localhost:5432/rails_test"
+      HTTP_PASSWORD: cddo-pwd
+      HTTP_USERNAME: cddo-team
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+
+      - name: Bundle
+        run: gem install bundler && bundle install
+
+      - name: Setup database
+        run: bundle exec rails db:schema:load
+
+      - name: Run tests
+        run: bundle exec rspec
+
+      - name: Run linter
+        run: bundle exec rubocop
```

Comment:
Does this block the build?

Reply:
I think it will! I thought it might be better that way than us just running the linter when we remember to 😄 If it is too annoying we can always remove it.